File: commontools/src/main/java/ir/mo/amin/hosseini/commontools/ObserverBase/ObserverList.java
package ir.mo.amin.hosseini.commontools.ObserverBase;

import ir.mo.amin.hosseini.commontools.Observers.NetworkObserverHandler;

/**
 * Created by Amin on 03/12/2017.
 */

public class ObserverList {
    public static NetworkObserverHandler getNetworkObserverHandler(){
        return NetworkObserverHandler.getInstance();
    }
}

File: calendarview/src/main/java/ir/mo/amin/hosseini/calendarview/Observers/ObserverList.kt
package ir.mo.amin.hosseini.calendarview.Observers

import ir.mo.amin.hosseini.calendarview.Helpers.Observers.ChangeDateObserverHandler
import ir.mo.amin.hosseini.calendarview.Helpers.Observers.ReminderObserverHandler
import ir.mo.amin.hosseini.commontools.ObserverBase.ObserverList

/**
 * Created by Amin on 11/7/2017.
 */
class ObserverList : ObserverList() {
    companion object {
        fun getChangeDataObserver() = ChangeDateObserverHandler.instance
        fun getReminderObserver() = ReminderObserverHandler.instance
    }
}